scripts/e2e-tests.sh

```diff
@@ -8,6 +10,10 @@ else
     TEST_NAMESPACE="openshift-marketplace"
 fi
 
+OPERATOR_SDK_BIN=${OPERATOR_SDK_BIN:=operator-sdk}
+export GO111MODULE=off
```

Comment:
The `GO111MODULE=off` is needed as Go 1.16 defaults this value to on.

This was discovered when updating the build root image to use Go 1.16:

```
INFO[2021-04-22T14:24:41Z] 2021-04-22 14:24:35 (78.7 MB/s) - '/tmp/operator-sdk' saved [56253389/56253389] 
INFO[2021-04-22T14:24:41Z]                                              
INFO[2021-04-22T14:24:41Z] Running operator-sdk test                    
INFO[2021-04-22T14:24:41Z] time="2021-04-22T14:24:35Z" level=info msg="Testing operator locally." 
INFO[2021-04-22T14:24:41Z] go: cannot find main module, but found Gopkg.lock in /go/src/github.com/operator-framework/operator-marketplace 
INFO[2021-04-22T14:24:41Z] 	to create a module there, run:              
INFO[2021-04-22T14:24:41Z] 	go mod init        
```

Reply:
Looking at https://github.com/operator-framework/operator-marketplace/commit/d973051f6cc7a24168f0463c95f4f9a1785f8907 we already updated the unit and builder script to set this, but not also the e2e test script(s).